typings/fast_deep_equal.d.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+declare const equal: (a: any, b: any) => boolean;
+export = equal;
```

Comment:
I would add a comment where this came from.

Looks like it is from one of these files?
https://github.com/epoberezkin/fast-deep-equal/commit/f0e3dc1a90f20fc5b3db52a6587cd41921ab49a4

and why we have to have it here and cannot use regular TypeScript without additional "cruft" and duplication for the next maintainer.

Reply:
Ok, I see this above:

```
/*
  TODO: Remove after https://github.com/epoberezkin/fast-deep-equal/pull/48
  is merged and fast-deep-equal version is bumped
*/
```

I think you are good!